libkbfs/ops.go

```diff
@@ -134,6 +178,17 @@ func (oc *OpCommon) getFinalPath() path {
 	return oc.finalPath
 }
 
+func (oc *OpCommon) checkUpdatesValid() error {
```

Comment:
Unfortunately, we need a special case for `checkValid()` in `createOp`, since it can have an empty `Dir` (for the root dir). A `checkAllUpdatesValid()` function would flag that as an error.